docs/administration.platforms.md

```diff
@@ -1,37 +1,70 @@
-- **Message**: The message that will be shown to all users. You can enter valid HTML in this field (except for the `<p>` tag). This is useful for e.g. supplying links to support pages.
-- **Severity**: The type of message (info, warning, critical).
-  - Info: for new features, upcoming services updates and general information that is not expected to disrupt platform operation
-  - Warning: best used to inform about partial platform outages or problems that may occur for some users
-  - Critical: appropriate in case of complete platform failures or impending platform downtimes
-- **Show From**: The earliest time the message will be shown. You can use this to create messages that will only be shown on at a later date.
-- **Show Until**: After this time the message will no longer be displayed.
+> All platform related maintenance like Landing Zones, Tenants, etc will be moved to the [Platform Control Plane](#platform-control-plane)
+> in future.
 
-At the bottom of the screen, a preview is shown. This can give you an idea of how the message is presented to your end-users.
+## Platform Creation
+
+You can integrate new meshPlatforms in self-service via a wizard in meshStack.
+
+![Platform Creation](assets/platform_maintenance/platform-creation.png)
+
+The following platform types are available for selection.
+
+![Platform Types](assets/platform_maintenance/platform-types.png)
+
+You have to assign a meshLocation to your meshPlatform and can provide additional provider information to the meshLocation if you
+would like to. This is the same information you can provide in the [Settings](#settings) tab of the Platform Control Plane.
+
+Additionally you can directly [restrict](#restrict-platform-access) your meshPlatform to certain meshCustomers to e.g. first test a new meshPlatform integration
+with a meshCustomer you have access to. This can prevent confusion of other users who may see the meshPlatform otherwise
+and would try to use it eventhough it is not working yet. Or you just have a dedicated Cloud Platform that shall only be available
+to certain meshCustomers in general.
+
+## Platform Control Plane
+
+The Platform Control Plane is the central entry point for any tasks or insights you want to do or get about your meshPlatform.
+
+![Platform Control Plane](assets/platform_maintenance/control-plane.png)
+
+### Financials
+
+In this tab you can get an overview about which kind of resources are being used in your Cloud Platform.
+
+### Settings
 
-## Restrict Platform Access
+This tab is mainly responsible for managing how your meshPlatform is shown and used inside meshStack. You can provide general information
+about your Platform like a display name or the Web Console Url meshStack will link to when the user wants to actually access the cloud platform.
 
```

Comment:
Maybe show a tip or smth: "For public cloud platforms you can leave the Web Console Url as is".